### DISCUSSION TITLE
Untuk mengubah persen (%)  ke dalam bentuk float

### DISCUSSION THREAD

**Discussion by snurulazizah** [teamalgoritma/community#154] [Q&A] [answered]
Apakah mesti kita hilangkan dulu tanda persen, dibagi 100, baru ubah ke type float64. Atau ada cara yang lebih ringkas/cepat ?

**Comment by yosiaazarya** [comment, marked as answer]
Adapun cara yang dapat dilakukan adalah dengan langkah sebagai berikut:
1. Menghapus tanda '%' terlebih dahulu dengan fungsi `.replace()`  -> `df['column'].str.replace('%','')`
2. Mengubah tipe data dengan fungsi `.astype()` -> `df['column'].astype('float64')`
